msg.py

```diff
@@ -42,7 +42,8 @@ def main():
     except KeyboardInterrupt:
         mqtt_client.disconnect()
         print('\ngoodbye !')
-
+    except ConnectionRefusedError:
```

Comment:
@Haider8 please leave a couple of lines before calling `if __name__ == '__main__'`. You might want to follow pep8 standards.